### DISCUSSION TITLE
Host public demos to increase user adoption

### DISCUSSION THREAD

**Discussion by SeanBannister** [bbc/peaks.js#525] [General]
Could I suggest that the demos at https://github.com/bbc/peaks.js/tree/master/demo/ be hosted somewhere to help drive user adoption of this project. I've been using wavesurfer for years because their public demos looked better but recently switched to peaks.js due to encountering performance issues with wavesurfer and was surprised at how much functionality is available. I'd been mislead to think peaks.js was limited due to the only public demo being [quite limited](https://waveform.prototyping.bbc.co.uk/).

**Comment by chrisn** [comment]
Welcome! This is a good point, the website could use some improvement, certainly.
